src/Compatibility/Core/src/iOS/Renderers/TabbedRenderer.cs

```diff
@@ -402,12 +402,19 @@ void UpdateBarTextColor()
 			if (!isDefaultColor)
 				_barTextColorWasSet = true;
 
-			var attributes = new UITextAttributes();
-
+			UIColor tabBarTextColor;
 			if (isDefaultColor)
-				attributes.TextColor = _defaultBarTextColor;
+				tabBarTextColor = _defaultBarTextColor;
 			else
-				attributes.TextColor = barTextColor.ToUIColor();
+				tabBarTextColor = barTextColor.ToUIColor();
+
+#if MACCATALYST
+			var attributes = new UIStringAttributes();
+			attributes.ForegroundColor = tabBarTextColor;
+#else
+			var attributes = new UITextAttributes();
+			attributes.TextColor = tabBarTextColor;
+#endif
```

Comment:
We can prob make this always use `UIStringAttributes` since UITextAttributes was added in iOS 6